### PR TITLE
refactor : integration data management account with mock api

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Changes
- add environment url mock api
- add plugins mock api
- change object instance to organization on constant management account 
- integration data management account using mock api

## Evidence
title: integration ldata management account with mock api
project: Jabar Super Apps
participants: @marsellavaleria19 @Ibwedagama @sheilaazhara @rizfirman

### CODE REVIEW

File: plugins/mockup-api.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// import { formatInTimeZone } from 'date-fns-tz'
```
Comment (Ibwedagama) [inline plugins/mockup-api.js:1]: is this code used? @marsellavaleria19

File: plugins/mockup-api.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export default function ({ $axios, $config }, inject) {
+  // Create a custom axios instance
+  const api = $axios.create({
+    baseURL: $config.urlMockApi,
+  })
+
+  // Inject to context as $api
```
Comment (Ibwedagama) [inline plugins/mockup-api.js:9]: this should be `inject to context as $mockApi`. no? @marsellavaleria19

**Review by Ibwedagama — APPROVED** [review]

**Review by sheilaazhara — APPROVED** [review]

File: plugins/mockup-api.js (hunk 1 above)
Comment (marsellavaleria19) [reply plugins/mockup-api.js]: It's not used @Ibwedagama . I'm sorry, I forgot to removed this. I fix it : https://github.com/jabardigitalservice/super-app-cms/pull/211/commits/e8e087d1deb361755e5156260e133dd770857db6

File: plugins/mockup-api.js (hunk 2 above)
Comment (marsellavaleria19) [reply plugins/mockup-api.js]: Yes, @Ibwedagama . It should be `inject context as $mockApi`. I forgot to change this. I fix it : https://github.com/jabardigitalservice/super-app-cms/pull/211/commits/b2a1fd7f12d6bbd2d919dff16978e0c9609e7d8c

**Review by rizfirman — APPROVED** [review]

**Pull request merged**